ocp_resources/reclaim_space_cron_job.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+from typing import Any, Dict, Optional
+from ocp_resources.resource import NamespacedResource, MissingRequiredArgumentError
+
+
+class ReclaimSpaceCronJob(NamespacedResource):
+    """
+    https://github.com/csi-addons/kubernetes-csi-addons/blob/main/apis/csiaddons/v1alpha1/reclaimspacecronjob_types.go
+    """
+
+    api_group = NamespacedResource.ApiGroup.CSIADDONS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        schedule: Optional[str] = "",
+        job_template: Dict[str, Any] | None = None,
```

Comment:
use Optional here as well?

Reply:
Job template is not optional for post/create calls. But for get() we don't need to pass it. I felt it would be misleading to mark it as optional. What do you suggest?